src/main/java/com/werfen/report/test/excel/ExcelComparator.java

```diff
@@ -0,0 +1,102 @@
+package com.werfen.report.test.excel;
+
+
+import org.apache.poi.openxml4j.exceptions.InvalidFormatException;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import static java.util.Objects.nonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class ExcelComparator {
```

Comment:
This is great, but we can't test that the code of this comparator is right unless we generate a release and use it in another project. Why not adding some minimal tests to check this comparison feature? They can rely on some given files and do things, like: 
* A file compared with himself passes the assert
* A file compared with another one fails the assert

Same applies for PDFs.